recipes/opengv/all/conanfile.py

```diff
@@ -55,25 +49,11 @@ def validate(self):
         if self.settings.compiler == "gcc" and self.options.shared:
             raise ConanInvalidConfiguration("Shared builds not supported with gcc since CCI errors out due to excessive memory usage.")
```

Comment:
I'll try to remove this once the build passes

Reply:
Let's do it in another PR. This PR passed affect much effort, so let's preserve its state.